week5/serving/fast_api.py

```diff
@@ -0,0 +1,73 @@
+from fastapi import FastAPI, File, UploadFile
```

Comment:
hm, I already saw file `[main.py](https://github.com/lyabomyr/Projector_ML_in_production_course/pull/25/files#diff-9fa53c587e8ce459bf9a8eb0926e7da1d1722984804857427f60fa9e54382d7b)`

is it duplicated file?

Reply:
file main.py I need for deployment to . deta  (https://fastapi.tiangolo.com/deployment/deta/)